docusaurus/docs/React/custom-code-examples/livestream-setup.mdx

```diff
@@ -0,0 +1,77 @@
+---
+id: livestream-setup
+sidebar_position: 22
+title: Livestream Best Practices
+---
+import MarginInspector from '../assets/margin-inspector.png'
+import MessageHeight from '../assets/message-height.png'
+
+Live stream chat UI poses several hard technological challenges since users can spend a few hours watching a certain event. For that period, the chat UI should be able to handle a constant stream of messages, reactions, and media attachments. 
+
+The React SDK offers a special component for that purpose - the `VirtualizedMessageList`. The component renders only the messages visible in the viewport and dynamically updates its contents as the user scrolls. 
+This decreases browser memory usage and does not degrade the performance of the page even if the user receives thousands of messages in a single session. 
+
+In this article, we will go through a best practices and tweaks for configuring the virtualized message list. 
```

Comment:
Consider changing to:
In this article, we will go through the best practices and tweaks for configuring the `VirtualizedMessageList` to ensure a successful event with lots of message traffic.